src/docquery/document.py

```diff
@@ -90,6 +104,14 @@ def apply_tesseract(*args, **kwargs):
     return transformers.apply_tesseract(*args, **kwargs)
 
 
+def apply_easyocr(*args, **kwargs):
+    global EASYOCR_READER
+    use_easyocr()
+    if not EASYOCR_READER:
+        EASYOCR_READER = easyocr.Reader(['en'])  # this needs to run only once to load the model into memory
```

Comment:
We should probably propagate the `device` argument here at some point (but I'm comfortable doing it in a follow up change). Do you mind leaving a TODO type comment about it?